envoy/npidds.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package pb.cilium;
+// package envoy.api.v2;
+
+import "api/discovery.proto";
+
+import "google/api/annotations.proto";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: Network policy identifier management and NPIDDS]
+
+// Each resource name is a host's IPv4 or IPv6 address, e.g. `10.1.2.3`.
```

Comment:
This is needed to separate between multiple Envoy instances? Alternatively the resource name could be something we explicitly configure on the policy enforcement filter, would be more predictable than one of the host's IP addresses.

Reply:
The way xDS protocols are designed, we need to pick what field to use as a resource name. Here, we have only 2 candidates: the IP address, or the policy ID. Only the IP address is usable as a resource name because multiple mappings can have the same policy ID, but never the same IP address.
Anyway it doesn't matter in our case because the filter will always query for all resources.

Reply:
As discussed offline it would be cleanest to use a configured filter string as the resource name.

Reply:
Done. Thanks!